.devcontainer/Dockerfile

```diff
@@ -3,12 +3,13 @@ ARG VARIANT="14"
 FROM mcr.microsoft.com/vscode/devcontainers/javascript-node:0-${VARIANT}
 
 # [Optional] Uncomment this section to install additional OS packages.
-# RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
-#     && apt-get -y install --no-install-recommends <your-package-list-here>
+RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
+	&& apt update \ 
+	&& apt install -y libasound2 libgbm-dev libgtk-3-0 libnss3 xvfb
```

Comment:
[F] These are the extra packages that are needed to make `xvfb-run` work inside the devcontainer.

Reply:
gotcha makes sense!